components/ws-manager-bridge/src/prometheus-metrics-exporter.ts

```diff
@@ -73,6 +74,12 @@ export class PrometheusMetricsExporter {
             labelNames: ["db_write", "workspace_cluster", "workspace_instance_type", "outcome"],
             buckets: prom.exponentialBuckets(2, 2, 8),
         });
+
+        this.prebuildsTotal = new prom.Counter({
```

Comment:
I created a duplicate one here because `prometheus-metrics.ts` isn't accessible from `ws-manager-bridge`. The metrics show up just fine as one in Prometheus, but please let me know if there is a better way 🙏🏽